portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,23 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+fetch('/data').then(response => response.json()).then((comment) => {
+	let doc = document.getElementById('comment-container');
+	for(x in comment){
+		doc.innerHTML += "<li>";
+		doc.innerHTML += "<h3>" + comment[x].name + "</h3>";
+		doc.innerHTML += "<p>" + comment[x].content + "</p>";
+		doc.innerHTML += "</li>";
+	}
+});
```

Comment:
It would be good to wrap this all up in a function, even if we just immediately invoke it from this file or in some place from the HTML. Most of the JavaScript you'll write will be in a function, which gives us more control over when it runs and makes it reusable. Right now, it runs ~sometime~ after the script is loaded by the browser... but it could happen any time, which isn't always what we want.